packages/react-component-library/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+export interface ButtonProps {
```

Comment:
consider extending ComponentWithClass

Reply:
Implementing on another branch and consolidating Button fixes there.